### ISSUE TITLE
Ability to Copy and Paste Passages

### ISSUE THREAD

**Issue by dylan-k** [klembot/twinejs#540]
I'd like to request a feature to select some passage(s) and do copy/paste to duplicate. Their duplicates' names might change to something like "Passage Name (copy)" to prevent confusion, but the content would be the same as the original(s).

This feature could make it easier to create passages from a template, to copy complex code/content,, or to create passages that are slight variations from each other.

(My apologies if this, my first feature request, is a duplicate, but when I looked I didn't see it.)

**Comment by piotrkov** [comment]
Yes please. Im also very much interested in this feature. It makes so much easier to cooperate with other devs and, as already mentioned, reusing code on multiple stories
